packages/workspace/src/generators/library/library.ts

```diff
@@ -75,7 +76,7 @@ export async function addLint(
   options: NormalizedSchema
 ): Promise<GeneratorCallback> {
   await ensurePackage(tree, '@nrwl/linter', nxVersion);
-  const { lintProjectGenerator } = require('@nrwl/linter');
+  const { lintProjectGenerator } = require('@nrwl/linter/generators');
```

Comment:
Why did these have to change?

Reply:
i think in packages/linter/index.ts, it also exports other utils functions. these utils may contain imports from typescript. so i just create a generators file and exports only generators.